docs/quart.rst

```diff
@@ -0,0 +1,71 @@
+Identity for Quart
+==================
+
+Prerequisite
+------------
+
+Create `a hello world web project in Quart <https://quart.palletsprojects.com/en/latest/tutorials/quickstart.html>`_.
+Here we assume the project's main file is named ``app.py``.
+
+
+Configuration
+--------------------------------
+
+#. Install dependency by ``pip install identity[quart]``
+
+#. Create an instance of the :py:class:`identity.quart.Auth` object,
+   and assign it to a global variable inside your ``app.py``::
+
+    import os
+
+    from quart import Quart
+    from identity.quart import Auth
+
+    app = Quart(__name__)
+
+    auth = Auth(
+        app,
+        os.getenv('CLIENT_ID'),
+        client_credential=os.getenv('CLIENT_SECRET'),
+        redirect_uri=os.getenv('REDIRECT_URI'),
+        ...,  # See below on how to feed in the authority url parameter
+        )
+
+   .. include:: auth.rst
+
+
+#. Add configuration to your ``app.py`` for Quart-session::
+
+    app.config['SESSION_TYPE'] = 'redis'
```

Comment:
```suggestion
    app.config['SESSION_TYPE'] = 'redis'  # pip install quart-session[redis]
```

This seems a good spot to mention the dependency requirement, which is otherwise buried deep in the setup.cfg

Reply:
Adding a note above that the package is auto installed, and added a link to its docs

Reply:
It does define extra, [here](https://github.com/kroketio/quart-session/blob/master/setup.py#L44).

Reply:
Ah you're right. I'm not used to reading setup.cfg files. Let me note that.

Reply:
I realized I installed redis independently and thats why I didnt realize there was an extra for it.

Reply:
Okay, made updates that should be clearer